src/middleware/github-server-app-middleware.ts

```diff
@@ -7,7 +7,7 @@ import { GITHUB_CLOUD_HOSTNAME } from "utils/get-github-client-config";
 
 export const GithubServerAppMiddleware = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 	const { jiraHost } = res.locals;
-	const { uuid } = req.params;
+	const uuid = req.params.uuid || req.query.uuid?.toString();
```

Comment:
Which url require this `uuid` to be in the query but not in the path? 
If any, is it better to update that url to include uuid in the path? Or is it not feasible?

Reply:
For the GH manage subscriptions page for GHES we navigate to /github/subscriptions/${installationId}/?gitHubAppId=${gitHubAppId}&uuid=${uuid} and for cloud we navigate to /github/subscriptions/${installationId}. I chose to go with a query string instead of params otherwise I would need to add 2 optional params here: GithubSubscriptionRouter.route("/:installationId") and figured it was better to leave that as is as opposed to the alternative of GithubSubscriptionRouter.route("/:installationId/:gitHubAppId?/:uuid?")

Reply:
I'd suggest change the frontend to navigate to `/github/:uuid/subscriptions/${installationId}` for managing ghe use cases. And then the current `gitHubServerMiddleware` will extract `gitHubAppId` from the uuid.